tests/unit/components/inputs/ReferenceInput.spec.js

```diff
@@ -0,0 +1,96 @@
+import { installQuasarPlugin } from '@quasar/quasar-app-extension-testing-unit-jest';
+import { mount } from '@vue/test-utils';
+import ReferenceInput from 'src/components/inputs/ReferenceInput';
+
+installQuasarPlugin();
+
+describe('Test component: StringInput', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(ReferenceInput, {
+      props: {
+        attribute: {
+          value: 'test',
+          name: 'attributeName',
+          definition: {
+            containerRef: 'reference',
+          },
+        },
+        plugin: {
+          name: 'pluginName',
+          components: [],
+          definitions: {
+            components: [{
+              type: 'reference',
+              icon: 'referenceIconName',
+            }],
+          },
+        },
+      },
+    });
+  });
+
+  describe('Test variables initialization', () => {
+    describe('Test prop: attribute', () => {
+      it('Should be an object containing name & value', () => {
```

Comment:
why not testing `attribute.definition` ? (same for plugin object)

Reply:
if I didn't make mistakes, I'm only testing the direct usages.
`attribute.value` become a `ref`, `attribute.name` is a `label` in the template and `plugin.name` is used in the `q-icon :name`.
The other used parts of attribute and plugin are check with the refs `options` and `iconName`.
So no need to duplicate. Please, notify me if I'm missing something.